docs/src/man/getting_started.md

```diff
@@ -558,6 +558,21 @@ Equivalently, the `in` function can be called with a single argument to create
 a function object that tests whether each value belongs to the subset
 (partial application of `in`): `df[in([1, 5, 601]).(df.A), :]`.
 
+!!! note
+
+    While `DataFrames` support indexing syntax that is very similar to
+    matrices, subsetting from a `DataFrame` will return a copy, not
+    a view.
+
+    `DataFrames` will only return a view when:
+
+    - a `!` is placed in the first indexing position (`df[!,:A]`, or `df[!, [:A, :B]]`),
```

Comment:
the first two options are not views - it is either a direct access to a vector stored in the data frame or creation of a new data frame without copying of the data.

Reply:
RIGHT! OK, then how about "`DataFrames` will not create a copy when:"

Reply:
this is OK, but DataFrames.jl not `DataFrames` (as we would talk about the functionality of the package).

Please always remember that `DataFrame` is only one concrete type we provide. We also provide `SubDataFrame` which is also a data frame.

Reply:
> Please always remember that DataFrame is only one concrete type we provide. We also provide SubDataFrame which is also a data frame.

right!